lib/cartopy/io/img_tiles.py

```diff
@@ -175,18 +179,23 @@ def _image_url(self, tile):
 
     def get_image(self, tile):
         if six.PY3:
-            from urllib.request import urlopen
+            from urllib.request import urlopen, Request, HTTPError, URLError
         else:
-            from urllib2 import urlopen
+            from urllib2 import urlopen, Request, HTTPError, URLError
 
         url = self._image_url(tile)
-
-        fh = urlopen(url)
-        im_data = six.BytesIO(fh.read())
-        fh.close()
-        img = Image.open(im_data)
-
-        img = img.convert(self.desired_tile_form)
+        try:
+            request = Request(url, headers={"user-agent": self.user_agent})
+            fh = urlopen(request)
+            im_data = six.BytesIO(fh.read())
+            fh.close()
+            img = Image.open(im_data)
+
+            img = img.convert(self.desired_tile_form)
+        except (HTTPError, URLError) as err:
+            print(err)
+            return (np.full((256, 256, 3), (252, 248, 248)),
```

Comment:
This causes a new warning:
```
numpy/core/numeric.py:294: FutureWarning: in the future, full(..., (252, 248, 248)) will return an array of dtype('int64')
    format(fill_value, array(fill_value).dtype), FutureWarning)
```
I think you need to specify an explicit dtype to silence this.

Also, if `desired_tile_form` is `RGBA` (or `ARGB`?), then it should return a different shape.